packages/eds-core-react/src/components/TopBar/TopBar.tsx

```diff
@@ -30,15 +36,19 @@ const StyledTopBar = styled.header(({ theme }) => {
   `
 })
 
-export const TopBar = forwardRef<HTMLElement, TopbarProps>(function TopBar(
-  { children, ...props },
+export const TopBar = forwardRef<HTMLDivElement, TopbarProps>(function TopBar(
+  { children, elevation = 'none', ...props },
   ref,
 ) {
   const { density } = useEds()
   const token = useToken({ density }, topbarToken)
+  const rest = {
+    ...props,
+    ref,
+  }
   return (
     <ThemeProvider theme={token}>
-      <StyledTopBar {...props} ref={ref}>
+      <StyledTopBar elevation={elevation} {...rest}>
```

Comment:
After changing this to use `Paper` it doesn't look like its a `header` element anymore. Maybe use the `as` prop to cast it here?

Reply:
I have now learned about `forwardedAs` since using `as` on an element also extended with styled(component) will not work